cli/mizarapi.py

```diff
@@ -26,14 +26,15 @@ def delete_vpc(self, name):
         logger.info("Delete a vpc!!!")
         self.delete_obj(name, "vpcs")
 
-    def create_net(self, name, ip, prefix, vpc, vni, bouncers=1):
+    def create_net(self, name, ip, prefix, vpc, vni, bouncers=1, external=False):
         logger.info("Creating subnet {}".format(name))
         spec = {
             "ip": ip,
             "prefix": prefix,
             "vni": vni,
             "vpc": vpc,
-            "bouncers": bouncers
+            "bouncers": bouncers,
+            "external": external
```

Comment:
Do we really need this? Why not just set the bouncer to the portal_host value?

Reply:
For the subnets in a local cluster, if any endpoints in the cluster want to talk to external clusters, it needs a gateway/portal host to forward their requests.  Portal host is  a cluster scope settings which can't be set up in subnets.

Reply:
@vinaykul I feel like this part needs a bit background design info to help with the code review. I'll set up a quick 30-min meeting this afternoon (or tomorrow if today doesn't work for you) for this.